src/modules/authorizer/extensions/AUT_EXT_VotingRoles_v1.sol

```diff
@@ -190,14 +187,9 @@ contract AUT_EXT_VotingRoles_v1 is IAUT_EXT_VotingRoles_v1, Module_v1 {
     //--------------------------------------------------------------------------
     // Configuration Functions
 
-    function setThreshold(uint newThreshold) external onlySelf {
-        // Revert if newThreshold higher than number of voters.
-        if (newThreshold > voterCount) {
-            revert Module__VotingRoleManager__UnreachableThreshold();
-        }
-
-        // Note that a threshold of zero is valid because a orchestrator can only be
-        // created by a voter anyway.
+    function setThreshold(uint newThreshold) public onlySelf {
```

Comment:
Curious why this function and `addVoter` visibilities have been set to public instead of external